<!-- example on exception handling -->
<!DOCTYPE html>
<html lang="en">
 <head>
	<title>Js Error</title>
 </head>
 <body>
    <h2>demo on try-catch blocks</h2> 
	<script>
		try{
			aler("welcome to  exception1");  
			alert("welcome to exception2"); 
		}catch(e){
			alert("Exception Occures" + "\n" + e + "\n" + e.name + "\n" + e.message);
			console.log("Exception Occures" + "\n" + e.name + "\n" + e.message);
		} 


 
	</script>
	<h3>end of page</h3>
 </body>
</html>

	